test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -225,5 +239,21 @@ void LoadBalancerFuzzBase::replay(
   }
 }
 
+void LoadBalancerFuzzBase::clearStaticHostsHealthFlags() {
+  // Have to clear the hosts health flags here - how do we know what hosts to clear?
+  // The only outstanding health flags set are those that are set from hosts being placed in
+  // degraded and excluded. Thus, use the priority set pointer to know which flags to clear.
```

Comment:
Could you also clear unconditionally?

Reply:
I think there is no unconditional flag clear, maybe a `clearFlagForTest` method or an ASSERT that the state matches expected?

Reply:
I think that my comment of "how do we know what hosts to clear?" is outdated. Since the only health flags that get set are those that are degraded and excluded, which will naturally be in that HostVector regardless? Thus making this work?